### PR TITLE
Pull request for sileht/doc

### PR DESCRIPTION
**Pull request by sileht**
doc: split measure sections
doc: add rate of change in aggregations list

### CODE REVIEW

File: doc/source/rest.j2 [hunk 1]
```diff
@@ -107,11 +110,17 @@ method. It is possible to request for any other method by specifying the
...
+They can be prefixed by "rate:" (like rate:last) to compute the rate of change
+before doing the aggregation.
+
 It's possible to provide the `granularity` argument to specify the granularity
 to retrieve, rather than all the granularities available:
 
 {{ scenarios['get-measures-granularity']['doc'] }}
 
+Measures transformations
+========================
```
Comment (chungg) [inline doc/source/rest.j2]: should we make transformations a subsection of measures

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 953652da85de6da28a34f02d3239192554c67f8d

**Comment by sileht** [comment]
Pull-request updated, HEAD is now b727bd178794e1a4966b8249554e79d7922210ad

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 87ccf4e6c424bc55c597c03fb1af721dbb832fd0

File: doc/source/rest.j2 [hunk 2]
```diff
@@ -308,20 +327,32 @@ some requests:
...
 
+Modification
+------------
+
 It's possible to modify a resource by re-uploading it partially with the
 modified fields:
 
 {{ scenarios['patch-resource']['doc'] }}
 
+Modification's history
```
Comment (jd) [inline doc/source/rest.j2]: History modification

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 54aa65480892303c5623dbec5eacff98d34cec72

**Comment by sileht** [comment]
Pull-request updated, HEAD is now b166253a5dcc8e0a4d3715be1c1968edf584e9a0

**Pull request merged**